easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-CrayGNU-18.07-python2.eb

```diff
@@ -0,0 +1,34 @@
+# contributed by Luca Marsella (CSCS), updated by Samuel Omlin (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'ANTLR'
+version = "2.7.7"
+versionsuffix = '-python2'
+
+homepage = 'http://www.antlr.org/'
+description = """ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
+ is a language tool that provides a framework for constructing recognizers,
+ compilers, and translators from grammatical descriptions containing
+ Java, C#, C++, or Python actions."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+
+source_urls = ['http://www.antlr2.org/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['%(name)s-%(version)s_includes.patch']
+
+
+dependencies = [
+    ('Java', '1.8.0_121', '', True),
```

Comment:
Do you know why we're installing a Java SDK instead of using the one from the system?

Maybe you could try using `java/jdk1.8.0_51`